packages/components/field-label/src/field-label.tsx

```diff
@@ -96,10 +106,13 @@ const FieldLabel = (props: TFieldLabelProps) => {
   return (
     <Constraints.Horizontal max={props.horizontalConstraint}>
       <Stack scale="xs">
-        <Inline alignItems="flexStart" scale="xs">
+        <Inline
+          alignItems={theme === 'default' ? 'flexStart' : 'center'}
+          scale="xs"
+        >
           <Text.Wrap>
             <Label
-              isBold={true}
+              isBold={theme === 'default' ? true : false}
```

Comment:
**nit**
```suggestion
              isBold={theme === 'default'}
```

Reply:
Improved with this commit - [e3aaf6b](https://github.com/commercetools/ui-kit/pull/2356/commits/e3aaf6b87520c5b8d055aafebf7bfdc1a69c3bcb)